src/test/java/EulerTwoTest.java

```diff
@@ -0,0 +1,84 @@
+import euler2.EulerTwo;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EulerTwoTest {
+    private EulerTwo eulerTwo;
+
+    @BeforeEach
+    void setUp() {
+        eulerTwo = new EulerTwo();
+    }
+
+    @Test
+    void shouldReturn10ForLimit9() {
+        //given
+        int limit = 9;
+
+        //when
+        int actualSum = eulerTwo.sumFibonacciEvenNumbers(limit);
+
+        //then
+        int expectedSum = 10;
+        assertEquals(expectedSum, actualSum);
+    }
+
+    @Test
+    void shouldNotReturn10ForLimit8() {
+        //given
+        int limit = 8;
+
+        //when
+        int actualSum = eulerTwo.sumFibonacciEvenNumbers(limit);
+
+        //then
+        int expectedSum = 10;
+        assertThat(actualSum, is(not(equalTo(expectedSum))));
```

Comment:
to jest bardzo dziwna asercja - "Zapewnij mnie że aktualna suma nie jest równa oczekiwanej sumie"

Reply:
Pamiętam że robiliśmy identyczną asercję na zajęciach dlatego dodałam. Tylko dane były inne. Moim celem było sprawdzenie wartości brzegowej - sąsiedniej do tej która dałaby sumę 10.W ten sposób chciałam upewnić się czy warunek w pętli był dobry i oczywiście nie był :) Więc asercja się przydała. Czy takie asercje są niezalecane? Czy raczej chodzi o nazewnictwo zmiennej "expectedSum"?

Reply:
Tak tutaj chodzi dokładnie o nazwę zmiennej `expectedSum`

Reply:
czy teraz jest lepiej? :) jakieś inne sugestie nazewnictwa dla takich przypadków?

Reply:
Tutaj raczej wpisywałbym to na sztywno do `equalTo` zamiast wynosić takie coś do zmiennej i jakoś to nazywać. Może ewentualnie nazwa `notExpectedSum`, ale to już trochę takie naciągane

Reply:
Poprawione